test/asynchronous/test_cursor.py

```diff
@@ -260,8 +259,11 @@ async def test_max_await_time_ms(self):
         self.assertEqual(99, listener.started_events[1].command["maxTimeMS"])
         listener.reset()
 
-        # Tailable_await with max_time_ms
-        await (await coll.find(cursor_type=CursorType.TAILABLE_AWAIT)).max_time_ms(99).to_list()
+        # Tailable_await with max_time_ms and make sure list() works on synchronous cursors
+        if _IS_SYNC:
+            list(await coll.find(cursor_type=CursorType.TAILABLE_AWAIT).max_time_ms(99))
```

Comment:
This is good but I'd also like to see a dedicated test for list() on a regular cursor, with multiple batches. Like "test_iteration_with_list"

Reply:
Explicitly testing the behavior on a synchronous cursor? Makes sense, will add.